PayGreen/paygreen-ui
# PGF-194: fix des composants DaTable (et sous composants) et Icons

## How to test?
- détails des fix à lire sur le ticket PGF-194
- beaucoup de fichiers updatés à cause des changements ds le thème
- yarn storybook : 
       - aller voir les composants DaTable, DaTableHead, DaTableHeadCell, DaTableRow
       - aller voir le composant Icon, tester le isActive et le hover quand isActive = true
- test en local de PG-UI sur le BO-Green aller sur la branche test-PGUI-PGF-194 et sur la page /users pour tester le tableau (intégration dans le header, les icons edit/see user dans les rows) attention c'est une branche de test ne regarder que le tableau users, elle ne contient pas les derniers updates de dashboard ou de réglages de la sidebar avec la page d'accueil :)
